src/extensions/selectable.ts

```diff
@@ -186,7 +186,7 @@ export function selectableNodes<T>(base: BaseAreaPlugin<Schemes, T>, core: Selec
     } else if (context.type === 'pointermove') {
       if (twitch !== null) twitch++
     } else if (context.type === 'pointerup') {
-      if (twitch !== null && twitch < 4) {
+      if (twitch !== null && twitch < 4 && context.data.event.button !==2) {
```

Comment:
anyone could use any other button to unselect nodes, so this would be considered a breaking change. It makes sense to make this customizable.